ERCS/erc-7291.md

```diff
@@ -0,0 +1,696 @@
+pragma solidity ^0.8.0;
+
+/// @notice Smart contracts MUST implement the ERC-165 `supportsInterface` function and signify support for the `PBMRC1_TokenReceiver` interface to accept callbacks.
+/// It is optional for a receiving smart contract to implement the `PBMRC1_TokenReceiver` interface
+/// @dev WARNING: Reentrancy guard procedure, Non delegate call, or the check-effects-interaction pattern must be adhere to when calling an external smart contract.
+/// The interface functions MUST only be called at the end of the `unwrap` function.
+interface PBMRC1_TokenReceiver {
+    /**
+        @notice Handles the callback from a PBM smart contract upon unwrapping
+        @dev An PBM smart contract MUST call this function on the token recipient contract, at the end of a `unwrap` if the
+        receiver smart contract supports type(PBMRC1_TokenReceiver).interfaceId
+        @param _operator  The address which initiated the transfer (either the address which previously owned the token or the address authorised to make transfers on the owner's behalf) (i.e. msg.sender)
+        @param _from      The address which previously owned the token
+        @param _id        The ID of the token being unwrapped
+        @param _value     The amount of tokens being transferred
+        @param _data      Additional data with no specified format
+        @return           `bytes4(keccak256("onPBMRC1Unwrap(address,address,uint256,uint256,bytes)"))`
+    */
+    function onPBMRC1Unwrap(address _operator, address _from, uint256 _id, uint256 _value, bytes calldata _data) external returns(bytes4);
+
+    /**
+        @notice Handles the callback from a PBM smart contract upon unwrapping a batch of tokens
+        @dev An PBM smart contract MUST call this function on the token recipient contract, at the end of a `unwrap` if the
+        receiver smart contract supports type(PBMRC1_TokenReceiver).interfaceId
+
+        @param _operator  The address which initiated the transfer (either the address which previously owned the token or the address authorised to make transfers on the owner's behalf) (i.e. msg.sender)
+        @param _from      The address which previously owned the token
+        @param _id        The ID of the token being unwrapped
+        @param _value     The amount of tokens being transferred
+        @param _data      Additional data with no specified format
+        @return           `bytes4(keccak256("onPBMRC1BatchUnwrap(address,address,uint256,uint256,bytes)"))`
+    */
+    function onPBMRC1BatchUnwrap(address _operator, address _from, uint256[] calldata _ids, uint256[] calldata _values, bytes calldata _data) external returns(bytes4);
+}
+
+```
+
+#### PBMRC2 - Non preloaded PBM Interface
+
+The **Non Preloaded** PBM extension is OPTIONAL for compliant smart contracts. This allows contracts to bind an underlying sovToken to the PBM at a later date instead of during a minting process.
+
+Compliant contract **MUST** implement the following interface:
```

Comment:
I'm assuming compliant here means compliant with the specific extension, and not with the whole document. Probably a good idea to disambiguate.